zoo-css-challenge/index.html

```diff
@@ -19,8 +20,9 @@
  		</nav>
  	</header>
  	<main>
- 		<section class="introduction">
+ 		<section class="introduction" style="color:white; line-height:30px">
```

Comment:
```suggestion
 		<section class="introduction">
```
We should add this CSS in the `.introduction` rule in the `style.css` file instead